terraform/workspace-variables/dev.tfvars

```diff
@@ -0,0 +1,13 @@
+# Platform
+environment = "dev"
+app_environment = "dev"
+
+# Gov.UK PaaS
+paas_api_url = "https://api.london.cloud.service.gov.uk"
```

Comment:
Into a secret ? Will it always have this value

Reply:
Lots will break if this changed 
https://github.com/search?q=https%3A%2F%2Fapi.london.cloud.service.gov.uk%2F&type=code

Reply:
Was just thinking of hiding it from the Russians 

They know it already I suppose